tests/MavenTests/testMzAligner.cpp

```diff
@@ -44,15 +35,65 @@ void TestMzAligner::testDoAlignment() {
     QVERIFY(true);
 }
 
-void TestMzAligner::testSaveFit(){
+void TestMzAligner::testObiWarp()
+{
+
+    MavenParameters* mavenparameters = new MavenParameters;
+    mavenparameters->compoundMassCutoffWindow->setMassCutoffAndType(5,"ppm");
+    mavenparameters->minQuality = 0.8;
+    mavenparameters->minIntensity = 10000;
+    mavenparameters->matchRtFlag = false;
+    mavenparameters->samples = samples->alignmentSamples;
+    mavenparameters->clsf = new ClassifierNeuralNet;
+    mavenparameters->clsf->loadModel("bin/default.model");
+
+    ObiParams params("cor", false, 2.0, 1.0, 0.20, 3.40, 0.0, 20.0, false, 0.60);
+    Aligner aligner;
+    aligner.alignWithObiWarp(mavenparameters->samples, &params);
+
+    DBS.loadCompoundCSVFile("bin/methods/KNOWNS.csv");
+    vector<Compound*> compounds = DBS.getCopoundsSubset("KNOWNS");
```

Comment:
I think it's high time we correct this typo "getCompoundSubset" :P

Reply:
Done